.circleci/config.yml

```diff
@@ -429,8 +429,85 @@ jobs:
       ENVOY_VERSIONS: "1.10.0"
     steps: *ENVOY_INTEGRATION_TEST_STEPS
 
+  # This job merges master into the 'current' branch (${CIRCLE_BRANCH}) specified
+  # in the branch filtering of the workflow
+  merge-master:
+    docker:
+      - image: *GOLANG_IMAGE
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+          - c6:96:98:82:dc:04:6c:39:dd:ac:83:05:e3:15:1c:98
+      - checkout
+      - run:
+          name: Merge Consul OSS master branch into current branch
+          command: |
+            set -eu -o pipefail
+
+            # Configure Git
+            git config --global user.email "hashicorp-ci@users.noreply.github.com"
+            git config --global user.name "hashicorp-ci"
+
+            # Fetch latest master
+            git fetch origin
+
+            # Create a merge branch to run tests on
+            git_merge_branch="ci/master-merge-$(date +%Y%m%d%H%M%S)"
+            git checkout -b "${git_merge_branch}"
+            latest_oss_commit="$(git rev-parse origin/master)"
+            git merge -m "Merge Consul OSS branch 'master' at commit ${latest_oss_commit}" origin/master
+            git push origin "${git_merge_branch}"
+            sleep 15  # Wait for merge branch to start CircleCI pipeline
+
+            # Wait for OSS merge branch CircleCI pipeline to finish
+            project_url="https://circleci.com/api/v1.1/project/github/hashicorp/consul/tree/${git_merge_branch}"
+            echo "Waiting for master merge branch CI pipeline to finish..."
+            builds=
+            unfinished_builds="will be populated in the until loop below"
+            min_waited=0
+            until [[ -z "${unfinished_builds}" ]]; do
+              builds="$(curl \
+                --header 'Accept: application/json' \
+                --show-error \
+                --silent \
+                --user "${CIRCLECI_API_TOKEN}:" \
```

Comment:
Is the trailing colon correct?

Reply:
That's probably a typo but actually I don't need a token for a public repo so I'll just remove this line.